qemu/tests/balloon_hotplug.py

```diff
@@ -36,6 +39,19 @@ def run_pm_test(pm_test, plug_type):
                               % (pm_test, plug_type), logging.info)
         utils_test.run_virt_sub_test(test, params, env, pm_test)
 
+    def enable_balloon_service(session):
+        """
+        Install balloon service and check its status in windows guests
+        :param session: shell Object
+        """
+        if params['os_type'] == 'windows':
+            balloon_test = BallooningTestWin(test, params, env)
+            balloon_test.configure_balloon_service(session)
+
+            output = balloon_test.operate_balloon_service(session, "status")
+            if not re.search("running", output.lower(), re.M):
+                test.error("Ballooon service status is not running")
+
```

Comment:
IIUC, if guest didn't install balloon service at beginning, L49 will install it. Then would it be running automatically after installation ? Or we need to call operate_balloon_service(session, "run") before L51?

Reply:
After installation, it will be in running status automatically